src/server/controllers/movie.js

```diff
@@ -15,6 +15,12 @@ const createMovie = async (req, res) => {
 
     const { title, description, runtimeMins } = req.body;
 
+    if(!title || !description || !runtimeMins) {
```

Comment:
Like this input verification.

Reply:
agreed. its something i should check for too!